### PR TITLE
bug/86cve6tmy: fixed pagination bug

### PR DESCRIPTION
**Pull request by NikkiShao**
fixed pagination paging bug due to not converting strings to numbers

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/pagination/Pagination.jsx [hunk 1]
```diff
@@ -51,6 +54,9 @@ export const Pagination = ({externalClassName, internalClassName, itemsPerPage,
     // splice only required items
     const currentItems = displayItems.slice(startIndex, endIndex);
 
+    // console.log("from:" + startIndex + " to:" + endIndex);
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pagination/Pagination.jsx:57]: Do we need these console.logs?

**Review by nethbtju — CHANGES_REQUESTED** [review]
CHANGES!!

File: behind-the-veil-siteroot/imports/ui/components/pagination/Pagination.jsx (hunk 1 above)
Comment (NikkiShao) [reply behind-the-veil-siteroot/imports/ui/components/pagination/Pagination.jsx]: nop

**Review by nethbtju — APPROVED** [review]
slay

**Pull request merged**